cmd/enmasse-controller-manager/main.go

```diff
@@ -58,8 +60,13 @@ func main() {
 
 	// register APIs
 
+	if err := api.Install(scheme.Scheme); err != nil {
+		log.Error(err, "Failed to register OpenShift schema")
```

Comment:
s/OpenShift/Kubernetes

Reply:
Nope, as this actually IS the OpenShift schema.

Reply:
Sorry, I didn't read the whole context of the change!